extensions/map-layers/src/mapLayers.ts

```diff
@@ -8,6 +8,7 @@ import { MapLayersUiItemsProvider } from "./ui/MapLayersUiItemsProvider";
 import { UiItemsManager, UiItemsProvider } from "@itwin/appui-abstract";
 import { FeatureInfoUiItemsProvider } from "./ui/FeatureInfoUiItemsProvider";
 import { MapFeatureInfoOptions, MapLayerOptions } from "./ui/Interfaces";
+import { WmsMapLayerFormat } from "./WmsMapLayerImageryFormats";
 
 export interface MapLayersConfig {
```

Comment:
@aruniverse We plan to add maplayers providers in this package... some consumers might want to initialize this package, but wont need the widget in their UI, so I guess all we need is a switch to skip 'MapLayersUiItemsProvider' ?